voltixApp/VoltixApp/Views/Vault/VaultPairDetailView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  VaultPairDetailView.swift
+//  VoltixApp
+//
+//  Created by Enrique Souza Soares on 10/04/2024.
+//
+import Foundation
+import SwiftUI
+
+struct VaultPairDetailView: View {
+    let vault: Vault
+    
+    @State var devicesInfo: [DeviceInfo] = []
+    @Environment(\.dismiss) var dismiss
+    
+    var body: some View {
+        ZStack {
+            Background()
+            view
+        }
+        .navigationBarBackButtonHidden(true)
+        .navigationTitle(NSLocalizedString("vaultDetailsTitle", comment: "View your vault details"))
+        .navigationBarTitleDisplayMode(.inline)
+        .toolbar {
+            ToolbarItem(placement: .topBarLeading) {
+                NavigationBackButton()
+            }
+        }
+        .onAppear {
+            self.devicesInfo = vault.signers.enumerated().map { index, signer in
+                DeviceInfo(Index: index, Signer: signer, PubKey: vault.keyshares[index].pubkey)
+            }
+        }
+    }
+    
+    var view: some View {
+        VStack {
+            content
+        }
+    }
+    
+    var content: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 12) {
+                
+                VaultPairDetailCell(title: NSLocalizedString("vaultName", comment: ""), description: vault.name).frame(maxWidth: .infinity, alignment: .leading)
+                
+                VaultPairDetailCell(title: "Device Name", description: Utils.getLocalDeviceIdentity()).frame(maxWidth: .infinity, alignment: .leading)
+                
+                VaultPairDetailCell(title: "ECDSA", description: vault.pubKeyECDSA).frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
put `ECDSA` , `EdDSA` to `localizable.strings` ?

Reply:
Yep. Done.